prosessering-jdbc/src/main/kotlin/no/nav/familie/prosessering/domene/TaskRepository.kt

```diff
@@ -9,4 +10,7 @@ import org.springframework.stereotype.Repository
 interface TaskRepository : ITaskRepostitory<Task> {
     @Query("""select t.type,t.status, count(*) as count from task t WHERE t.status in ('UBEHANDLET', 'BEHANDLER', 'PLUKKET', 'KLAR_TIL_PLUKK') GROUP BY t.type, t.status""")
     override fun countOpenTasks(): List<AntallÅpneTask>
+
+    @Query("""select distinct t.id from task t join task_logg tl on t.id = tl.task_id WHERE t.status = 'FERDIG' and tl.type = 'FEILET'""")
```

Comment:
Prøvde med select distinct t her, men da fekk eg denne kryptiske feilmeldinga i ba-sak:
"An empty SqlIdentifier can't be used in to create column names null-identifier", som ser ut til å treffe på koplinga frå task til task_logg. Prøvde litt googling og eit par minutts fiksing, men det var ikkje rett fram.

Så det vil jo gi nokre ekstra kall, men eg trur ikkje det er så mange at det blir problematisk. Men her kan det vera noko eg ikkje tenkjer på.

Reply:
Problemet eg støtte på kom visst av at eg brukte `select distinct t from...` og ikkje `select distinct t.* from...`